### ISSUE TITLE
How to add pt-pt with pt-br already as portuguese?

### ISSUE THREAD

**Issue by TarantinoMariachi** [filegator/filegator#191]
portuguese file has the portuguese brazillian language how can I add the mother language portuguese european?

**Comment by alcalbg** [comment]
Hi,

Huh, I'm not sure. Maybe we can have portuguese_pt and portuguese_br?

**Comment by Leinad4Mind** [comment]
I believe that could work. Would be better to use the ISOs.
Like pt_pt, pt_br, en_us. Etc.

**Comment by alcalbg** [comment]
I agree, but that would introduce a breaking change for all current installations since they have full language name in their config files.
